pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -682,44 +682,44 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			},
 			Entry("and accept input with virtio bus",
 				v1.Input{
-					Type: "tablet",
+					Type: v1.InputTypeTablet,
 					Name: "tablet0",
-					Bus:  "virtio",
+					Bus:  v1.InputBusVirtio,
 				}, 0, []string{}, "Expect no errors"),
 			Entry("and accept input with usb bus",
 				v1.Input{
-					Type: "tablet",
+					Type: v1.InputTypeTablet,
 					Name: "tablet0",
-					Bus:  "usb",
+					Bus:  v1.InputBusUSB,
 				}, 0, []string{}, "Expect no errors"),
 			Entry("and accept input without bus",
 				v1.Input{
-					Type: "tablet",
+					Type: v1.InputTypeTablet,
 					Name: "tablet0",
 				}, 0, []string{}, "Expect no errors"),
 			Entry("and reject input with ps2 bus",
 				v1.Input{
-					Type: "tablet",
+					Type: v1.InputTypeTablet,
 					Name: "tablet0",
-					Bus:  "ps2",
+					Bus:  v1.InputBus("ps2"),
 				}, 1, []string{"fake.domain.devices.inputs[0].bus"}, "Expect bus error"),
 			Entry("and reject input with keyboard type and virtio bus",
 				v1.Input{
-					Type: "keyboard",
+					Type: v1.InputType("keyboard"),
```

Comment:
This can be `InputTypeKeyboard` constant.

Reply:
Done